ksp_plugin_adapter/window_renderer.cs

```diff
@@ -114,7 +114,7 @@ internal abstract class BaseWindowRenderer : ScalingRenderer, IConfigNode {
     var old_skin = UnityEngine.GUI.skin;
     if (skin_ == null) {
       UnityEngine.GUI.skin = null;
-      skin_ = MakeSkin(UnityEngine.GUI.skin);
+      skin_ = MakeSkin(UnityEngine.GUI.skin!);
```

Comment:
This is a case where we would really need to annotate the engine declarations:
the property `UnityEngine.GUI.skin` is `UnityEngine.GUISkin`, not `UnityEngine.GUISkin?`, but it is `[AllowNull]` (`set` accepts null, `get` never returns null).